bin/run_data_task.py

```diff
@@ -53,13 +53,11 @@ def main():
         request = env["request"]
         settings = env["registry"].settings
 
-        Regions.set_region(settings["h_authority"], settings["region_code"])
-
         scripts = data_tasks.from_dir(
             task_dir=TASK_ROOT / args.task,
             template_vars={
                 "db_user": parse_dsn(settings["database_url"].strip())["user"],
-                "region": Regions.get_region(),
+                "region": request.find_service(RegionService).get(),
```

Comment:
This is actually the only place where `RegionService` is used. We could replace this with a call to the `get_current_region()` function from the models (perhaps move it to `models.Region.get()`) and then `RegionService` wouldn't be needed.

On the other hand, I feel like `RegionService` is the right way to do this and `get_current_region()` is a necessary hack. Especially if we ever end up addin more code that gets the region, I'd rather it used the service.

So maybe leave it in?

I could go either way.